src/main/java/se/bjurr/prnfb/service/ButtonsService.java

```diff
@@ -126,19 +127,25 @@ private boolean isVisibleOnRepository(PrnfbButton button, Repository repository)
  }
 
  @VisibleForTesting
- void doHandlePressed(UUID buttonUuid, ClientKeyStore clientKeyStore, boolean shouldAcceptAnyCertificate,
+ Map<String, HttpResponse> doHandlePressed(UUID buttonUuid, ClientKeyStore clientKeyStore, boolean shouldAcceptAnyCertificate,
```

Comment:
Happy to switch it to be keyed off the UUID of the Notification object, though I would like to include the name there just to avoid (for now) having to make another request per triggered notification. The UI side code is already complicated as it is :)